lib/ueberauth/strategy/cognito/jwt_utilities.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Ueberauth.Strategy.Cognito.JwtUtilities do
```

Comment:
Since all the functions here are for verifying a token, you could give this module a more specific name.

Reply:
We opted to split out a separate `Utilities` module to handle the shared URL logic and rename this one to `JwtVerifier`.